src/output-json-stats.c

```diff
@@ -68,6 +68,8 @@ static json_t *OutputStats2Json(json_t *js, const char *key)
     const char *dot = index(key, '.');
     if (dot == NULL)
         return NULL;
+    if (*(dot + 1) == '.' && *(dot + 2) != '\0')
```

Comment:
is this a safe access? It looks a bit fishy to me. OTOH if we have dot, *(dot+1) would exist (could be \0) and if it is not \0 *(dot+2) should also be valid. Still I'd be happier if this would include strlen(dot) > 2

Reply:
That was my thinking as well (indeed I spent quite some time pondering about these cases) but I wouldn't mind including additional checks.